tests/Fixer/Basic/NonPrintableCharacterFixerTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Basic;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author Ivan Boprzenkov <ivan.borzenkov@gmail.com>
+ *
+ * @internal
+ */
+final class NonPrintableCharacterFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideExamples
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideExamples()
+    {
+        return array(
```

Comment:
Could you please add more test cases, with invisible characters in comments for example?